static/app/components/replays/configureReplayCard.tsx

```diff
@@ -54,6 +54,11 @@ function ResourceButtons() {
         subtitle={t('Unmask text (****) and unblock media (img, svg, video, etc.)')}
         link="https://docs.sentry.io/platforms/javascript/session-replay/privacy/#privacy-configuration"
       />
+      <Resource
+        title={t('Identify Users')}
+        subtitle={t('Identify your users through a specific format, such as email.')}
```

Comment:
Not sure "format" is the right word here...

Maybe attribute? property? Don't love those either...

Reply:
hm @jas-kas do you have a pref? attribute & property are both fine w me

Reply:
Attribute sounds good to me

Reply:
thanks @Jesse-Box ! i'll update